canopy/design-system/package.json

```diff
@@ -2,14 +2,24 @@
   "name": "canopy-design-system",
   "version": "0.1.0",
   "private": true,
+  "main": "lib/components.min.js",
```

Comment:
Can you add some info to the readme on how to use these scripts?

Reply:
updated